include/certifier_utilities.h

```diff
@@ -34,10 +34,13 @@ namespace utilities {
 const int block_size = 16;
 const int num_bits_in_byte = 8;
 
-bool write_file(const string &file_name, int size, byte *data);
-int  file_size(const string &file_name);
+int file_size(const string &file_name);
+
 bool read_file(const string &file_name, int *size, byte *data);
+bool write_file(const string &file_name, int size, byte *data);
```

Comment:
In the proposal doc we reviewed, there is a mention of reworking read() / write() interfaces.

I see these new ones being added to this file, but the ones in `certifier_framework.h` still seem to remain unchanged; see this triplet below:

```
382   int  read(string *out);
383   int  read(int size, byte *b);
384   int  write(int size, byte *b);
```
Qs. is: Are these new interfaces a substitute for the existing ones? And, there is a mention in the doc to add some extended interfaces. Is there a plan to go back nand address the issues w.r.t above 3 interfaces which we discussed earlier?